example/storybook/stories/BrandConfig/BrandConfig.stories.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react-native';
+import { BrandConfigProvider } from 'src';
+import { View, ViewStyle } from 'react-native';
+import { text, object } from '@storybook/addon-knobs';
+import { Theme } from 'src/components/BrandConfigProvider/Theme';
+import { Styles } from 'src/components/BrandConfigProvider/Styles';
+import { Examples } from './Examples';
+import { TextStyles } from 'src/components/Text/TextStyles';
+
+const defaultTextContent = 'Example text content.';
+
+const centerView: ViewStyle = {
+  flex: 1,
+  justifyContent: 'center',
+  alignItems: 'center',
+};
+
+storiesOf('Brand Config', module)
+  .addDecorator((story) => <View style={centerView}>{story()}</View>)
+
+  .add('Basic', () => {
+    const content = text('Text Content', defaultTextContent);
+
+    return (
+      <BrandConfigProvider>
+        <Examples content={content} />
+      </BrandConfigProvider>
+    );
+  })
+
+  .add('Custom theme', () => {
+    const content = text('Text Content', defaultTextContent);
+
+    const themeConfig = object('Theme', {
+      colors: {
+        text: 'blue',
+      },
+    });
+
+    const theme = new Theme(themeConfig);
+
+    return (
+      <BrandConfigProvider theme={theme}>
+        <Examples content={content} />
+      </BrandConfigProvider>
+    );
+  })
```

Comment:
The Theme object can be injected directly into the provider if no component customization is needed.